.github/workflows/post-size-cmp.yml

```diff
@@ -0,0 +1,115 @@
+---
+name: Post Comment for Size Comparison
+
+on:
+  workflow_run:
+    workflows: ["Size Comparison"]
+    types:
+      - completed
+
+jobs:
+  size-cmp:
+    name: Post Comment on Pull Request
+    runs-on: ubuntu-latest
+
+    steps:
+      - if: github.event.workflow_run.event == 'pull_request'
+        name: Download Artifact
+        uses: Legit-Labs/action-download-artifact@v2
+        with:
+          github_token: "${{ secrets.GITHUB_TOKEN }}"
+          workflow: size-cmp.yml
+          run_id: ${{ github.event.workflow_run.id }}
+          name: size-cmp-info
+          path: "size-cmp-info/"
+
+      - name: Make pull request comment
+        run: |
+          from typing import Dict, List, Optional
+
+          import os
+          import json
+
+          with open("size-cmp-info/.SIZE_CMP_INFO") as f:
+              content = json.loads(f.read())
+
+          joined_sizes = content["sizes"]
+
+          lines: List[str] = []
+
+          lines.append("### Size Comparison")
+          lines.append("")
+          lines.append("| examples | master (KB) | pull request (KB) | diff |")
+          lines.append("| --- | --- | --- | --- | ")
+
+          for (i, sizes) in joined_sizes:
+              (master_size, pr_size) = sizes
+              diff: Optional[int] = None
+
+              if master_size is not None and pr_size is not None:
+                  diff = pr_size - master_size
+
+              master_size_str = "N/A" if master_size is None else (
+                  f"{master_size / 1024:.3f}" if master_size != 0 else "0"
+              )
+              pr_size_str = "N/A" if pr_size is None else (
+                  f"{pr_size / 1024:.3f}" if pr_size != 0 else "0"
+              )
+              diff_str = "N/A" if diff is None else (
+                  f"{diff / 1024:.3f}" if diff < 0 else (
+                      f"+{diff / 1024:.3f}" if diff > 0 else "0"
+                  )
+              )
+              lines.append(f"| {i} | {master_size_str} | {pr_size_str} | {diff_str} |")
+
+          output = "\n".join(lines)
+
+          with open(os.environ["GITHUB_ENV"], "a+") as f:
+              f.write(f"YEW_EXAMPLE_SIZES={json.dumps(output)}\n")
+              f.write(f"PR_NUMBER={content["issue_number"]}\n")
+
```

Comment:
I'd rather have these parts in proper files than inline in yaml files, but it's fine for now.

Reply:
This could go in the ci directory

Reply:
@futursolo, do you want to make this change? If not, please go ahead and merge this